internal/helm/registry/auth.go

```diff
@@ -68,3 +69,22 @@ func LoginOptionFromSecret(registryURL string, secret corev1.Secret) (registry.L
 	}
 	return registry.LoginOptBasicAuth(username, password), nil
 }
+
+// OIDCAdaptHelper adapt crane authenticator to oras credential function
```

Comment:
```suggestion
// OIDCAdaptHelper returns an ORAS credentials callback configured with the authorization data
// from the given authn authenticator. This allows for example to make use of credential helpers from
// cloud providers.
// Ref: https://github.com/google/go-containerregistry/tree/main/pkg/authn
```

Reply:
I would maybe also add a tiny test to ensure the credentials are actually propagated to the callback.